Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterCreateEditWidget.cpp

```diff
@@ -25,51 +26,36 @@
 
 namespace EMStudio
 {
-    int ParameterCreateEditDialog::s_parameterEditorMinWidth = 300;
+    const int ParameterCreateEditWidget::s_parameterEditorMinWidth = 300;
 
-    ParameterCreateEditDialog::ParameterCreateEditDialog(AnimGraphPlugin* plugin, QWidget* parent, const EMotionFX::Parameter* editParameter)
-        : QDialog(parent)
+    ParameterCreateEditWidget::ParameterCreateEditWidget(AnimGraphPlugin* plugin, QWidget* parent)
+        : QWidget(parent)
         , m_plugin(plugin)
         , m_valueTypeCombo(nullptr)
         , m_valueParameterEditor(nullptr)
     {
-        setObjectName("ParameterCreateEditDialog");
+        setObjectName("ParameterCreateEditWidget");
+        setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Maximum);
 
-        if (!editParameter)
-        {
-            setWindowTitle("Create Parameter");
-            m_createButton = new QPushButton("Create", this);
-        }
-        else
-        {
-            m_parameter.reset(MCore::ReflectionSerializer::Clone(editParameter));
-            setWindowTitle("Edit Parameter");
-            m_createButton = new QPushButton("Apply", this);
-            m_originalName = editParameter->GetName();
-        }
-        m_createButton->setObjectName("EMFX.ParameterCreateEditDialog.CreateApplyButton");
+        m_createButton = new QPushButton("Create", this);
```

Comment:
Nit: Any user-facing strings should be wrapped in `tr` so we can handle translation in the future. (Here and several other spots in the PR)

```suggestion
        m_createButton = new QPushButton(tr("Create"), this);
```

Reply:
Agreed